deploy/k8s/postgres-clair-deployment.yaml

```diff
@@ -16,18 +16,21 @@ spec:
       labels:
         app: postgres-clair
     spec:
+      securityContext:
+        runAsUser: 0 
+        fsGroup: 0
       containers:
       - env:
-        - name: POSTGRESQL_USER
+        - name: POSTGRES_USER
           # Set the username for the Clair postgres database (clair by default)
           value: clair
-        - name: POSTGRESQL_DATABASE
+        - name: POSTGRES_DATABASE
           # Set the name of the Clair postgres database
           value: clair
-        - name: POSTGRESQL_PASSWORD
+        - name: POSTGRES_PASSWORD
           # Set the password for the Clair postgress user
           value: test123
-        image: registry.access.redhat.com/rhscl/postgresql-10-rhel7:1-35
```

Comment:
I'd prefer to continue using the Red Hat postgresql, what is the reason for changing?

Reply:
Short answer: It didn't work.  I don't remember why off the top of my head, but I found that using the more generic postgres image didn't error out.  When it's run in your environment, do you see any issues with it, or does it work fine?

Reply:
@thomasmckay I put the image definition back.  Let me know if you have any other comments.